src/Altinn.App.Core/Internal/Expressions/ExpressionValue.cs

```diff
@@ -268,7 +337,6 @@ internal class ExpressionTypeUnionConverter : JsonConverter<ExpressionValue>
     /// <inheritdoc />
     public override ExpressionValue Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
     {
-        reader.Read();
```

Comment:
Was this call moved somewhere or was it actually not needed? :O

Reply:
It is just plain wrong, and made it ard to write tests for the code. You don't call `.Read()` when reading primitive types (only on object/array).

I might not actually need this converter (yet), because I always use `Expression` for serialisation, but with proper test coverage I think it makes sense to have.